### PR TITLE
Party new

### LINKED ISSUE #6: New Viewing Party Page
As an authenticated user,
When I visit the new viewing party page,
I should see a form with the following:
- [x] Movie Title (that is un-editable)
- [x] Duration of Party with a default value of movie runtime in minutes
- [x] When: field to select date
- [x] Start Time: field to select time
- [x] Checkboxes next to each friend (if user has friends)
- [x]  Button to create a party

**Details** When the party is created, the authenticated user should be redirected back to the dashboard where the new event is shown. The event should also be seen by any friends that were invited when they log in.

### PR DESCRIPTION
**Pull request by araspaas**
Add feature to allow a registered user to create a viewing party

- Add tests for creating a new viewing party
- Add a user being logged in for testing
- Add a form for creating a new viewing party
- Add create action in party controller and party params method

closes #6

### CODE REVIEW

File: app/controllers/party_controller.rb [hunk 1]
```diff
@@ -5,11 +5,32 @@ def new
...
-      Movie.create!({title: params[:title], duration: params[:duration], api_id: params[:api_id]})
+      @movie = Movie.create!({title: params[:title], duration: params[:duration], api_id: params[:api_id]})
     end
   end
 
-  # def create
-  #
-  # end
+  def create
+    party = Party.new(party_params)
```
Comment (smcunning) [inline app/controllers/party_controller.rb:13]: Such a messy action! Can't wait to simplify in refactor

File: app/controllers/party_controller.rb [hunk 2]
```diff
@@ -5,11 +5,32 @@ def new
...
     else
-      Movie.create!({title: params[:title], duration: params[:duration], api_id: params[:api_id]})
+      @movie = Movie.create!({title: params[:title], duration: params[:duration], api_id: params[:api_id]})
     end
   end
 
-  # def create
-  #
-  # end
+  def create
```
Comment (KieraAllen) [inline app/controllers/party_controller.rb:12]: This could be part of our refactor work

File: app/views/party/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  <%= f.date_field :date, value: f.object.try(:strftime,"%m/%d/%Y"), class: 'form-control' %>
+
+  <%= f.label :start_time %>
+  <%= f.text_field :start_time, placeholder: "12:00" %>
+
+  <% if current_user.friends.empty? %>
+    <p>You don't have any friends to add to the party</p>
+  <% else %>
+    <span>Include: </span>
+    <%= collection_check_boxes :friends, :ids, current_user.friends, :id, :name %><br/>
```
Comment (smcunning) [inline app/views/party/new.html.erb:24]: Such a hard won line of code right here!

**Review by smcunning — APPROVED** [review]

File: spec/features/movies/show_spec.rb [hunk 4]
```diff
@@ -23,6 +23,11 @@
 
 describe "As a user on the movies show page" do
   before :each do
+    @user = User.create!(name: "Tim", email: "tim@gmail.com", password: "test")
```
Comment (KieraAllen) [inline spec/features/movies/show_spec.rb:26]: Nice catch @smcunning spotting that we didn't have another user logging in.

**Review by KieraAllen — APPROVED** [review]

**Pull request merged**